server/src/internalClusterTest/java/org/opensearch/indices/store/IndicesStoreIntegrationIT.java

```diff
@@ -168,6 +168,97 @@ public void testIndexCleanup() throws Exception {
 
     }
 
+    /**
+     * Verify files are removed even if there is another relocation is in progress.
+     * Scenario in IT:
+     * Two relocation is in progress, once first relocation is completes it removes files from source node
```

Comment:
Two relocations are in progress.  Removes files from source node once first relocation completes without waiting for second relocation to finish

Reply:
Updated.